lib/menu.rb

```diff
@@ -0,0 +1,35 @@
+class Menu
+  def initialize
+    @menu = {}
+  end
+
+  def add dish, price
+    @menu[dish] = price
+  end
+
+  def show # abstract this to a showable module shared by order and receipt classes? takes an { item: price } hash and lists it on new lines...
```

Comment:
I still think this should be called `print` or `pretty_print` and what would be amazing is if it took a `Printer` that decided to print it however it liked.